commerce/domain/pricing/cart/PricingBasicCalculator/PricingCalculatorSampleTest.cls

```diff
@@ -0,0 +1,204 @@
+/**
+* @description A Sample unit test for PricingCalculatorSample.
+*/
+@IsTest
```

Comment:
Do we have at least `75%` code coverage for `PricingCalculatorSample` with this test?

Reply:
i got 67% coverage. To get more coverage i need to change the code logic a bit. But that part is mostly covered by Pavel in his sample, so i stopped at 67.

Reply:
Could we please reach 75? This is public sample and it gotta be perfect 🙂

Reply:
code coverage is 82% now.